### ISSUE TITLE
Possible Test error on keyboard focus order question 4.F

### ISSUE THREAD

**Issue by Access508** [Section508Coordinators/DHSA-TT-200-A#440]
## Issue Type (put a capital X between the brackets to indicate issue type)
- [ X ] System Bug (the course doesn't function properly)
- [ ] Typo/grammar
- [ ] Incorrect content
- [ ] 508 issue (something is not accessible according to our own test process)
- [ ] Other, specify...[replace this text with your issue type description]

## Description
Test page had 3 focus (tab) order errors.  I checked item 4.F as failed.  Then Instructions said to select the Last element with correct focus order.  I did this and submitted (on 3 separate tries).  Each time my answer was submitted I received an error stating that 4.F could fail based on a non-keyboard-operable item, or a keyboard trap, or an incorrect order.  The instructions only accommodated the last situation, of incorrect order.

## Location
https://exams.section508testing.net/dhsv5/scormapi/lesson1.php?SCOInstanceID=36875&studentName=Bruce%20Ladino&ltiUserId=7051&studentEmail=Bladino508@yahoo.com&examId=4
TT-200-A, Topic 4, F

## Actual Behavior / Observation
Unable to get the answer correct to move on with the course

## Expected Behavior / Recommendation
Instructions in the failure elements need to match the criteria being sought:
Instructions -- check one box before the first error
Fail Response -- 4F can apply to non-keyboard-accessible, keyboard trap (for which there is another test (4C), and improper order.

QUESTION -- When filling out the failure element, should I ignore the first focus order issue because it was tested in 4C, and enter the number of the first following element before the focus order problem?

## Steps to Reproduce the Issue (if applicable)
answered the question according to directions, and submitted normally.
